astropy/coordinates/representation.py

```diff
@@ -319,6 +319,8 @@ def to_cartesian(self):
         """Convert the representation to its Cartesian form.
 
         Note that any differentials get dropped.
+        Also, orientation information at the origin is NOT preserved by
+        conversions through Cartesian coordinates.
```

Comment:
Maybe add an example?
```suggestion
        Also note that orientation information at the origin is *not* preserved by
        conversions through Cartesian coordinates. For example, transforming 
        an angular position defined at distance=0 through cartesian coordinates
        and back will lose the original angular coordinates::
        
            >>> import astropy.units as u
            >>> import astropy.coordinates as coord
            >>> rep = coord.SphericalRepresentation(
            ...     lon=15*u.deg, 
            ...     lat=-11*u.deg,
            ...     distance=0*u.pc)
            >>> rep.to_cartesian().represent_as(coord.SphericalRepresentation)
            <SphericalRepresentation (lon, lat, distance) in (rad, rad, pc)
                (0., 0., 0.)>
```